dotnet/src/Plugins/Plugins.MsGraph/Connectors/Client/MsGraphClientLoggingHandler.cs

```diff
@@ -65,13 +65,26 @@ private void LogHttpMessage(HttpHeaders headers, Uri? uri, string prefix)
     {
         if (this._logger.IsEnabled(LogLevel.Debug))
         {
-            StringBuilder message = new();
-            message.AppendLine($"{prefix} {uri}");
+            var message = new StringBuilder().Append(prefix).Append(' ').Append(uri).AppendLine();
```

Comment:
```suggestion
            var message = new StringBuilder().Append(prefix).Append(' ').AppendLine(uri);
```

Reply:
There's no AppendLine overload that works with a Uri object.